test/integration/artist_can_delete_their_items_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.